src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -2818,4 +2818,19 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Convert_to_record" xml:space="preserve">
     <value>Convert to record</value>
   </data>
+  <data name="Implementing_a_record_positional_parameter_0_as_read_only_will_prevent_the_debug_session_from_continuing" xml:space="preserve">
+    <value>Implementing a record positional parameter '{0}' as read only will prevent the debug session from continuing,</value>
+  </data>
+  <data name="Implementing_a_record_positional_parameter_0_with_a_set_accessor_will_prevent_the_debug_session_from_continuing" xml:space="preserve">
+    <value>Implementing a record positional parameter '{0}' with a set accessor will prevent the debug session from continuing.</value>
+  </data>
+  <data name="Adding_a_positional_parameter_to_a_record_will_prevent_the_debug_session_from_continuing" xml:space="preserve">
+    <value>Adding a positional parameter to a record will prevent the debug session from continuing.</value>
+  </data>
+  <data name="Deleting_a_positional_parameter_from_a_record_will_prevent_the_debug_session_from_continuing" xml:space="preserve">
+    <value>Deleting a positional parameter from a record will prevent the debug session from continuing.</value>
+  </data>
+  <data name="Explicitly_implemented_methods_of_records_must_have_parameter_names_that_match_the_compiler_generated_equivalent_0" xml:space="preserve">
+    <value>Explicitly implemented methods of records must have parameter names that match the compiler generated equivalent '{0}'</value>
```

Comment:
@jcouv Is "positional parameter" the correct wording to use here? Or "primary constructor parameter"? or something else?